core/pta/widevine.c

```diff
@@ -0,0 +1,188 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2023, The ChromiumOS Authors
+ */
+
+#include <compiler.h>
+#include <initcall.h>
+#include <kernel/boot.h>
+#include <kernel/dt.h>
+#include <kernel/pseudo_ta.h>
+#include <libfdt.h>
+#include <malloc.h>
+#include <mm/tee_mm.h>
+#include <mm/tee_pager.h>
+#include <pta_widevine.h>
+#include <stdio.h>
+#include <string.h>
+#include <string_ext.h>
+#include <tee_api.h>
+#include <trace.h>
+#include <util.h>
```

Comment:
can remove _boot.h_, _malloc.h_, _tee_mm.h_, _tee_pager.h_, _stdio.h_, _string_ext.h_ and _trace.h_.
Need to include _stdint.h_, _ts_manager.h_, and _tee_te_manager.h_.